src/NumericalAlgorithms/DiscontinuousGalerkin/Actions/FluxCommunication.hpp

```diff
@@ -72,13 +72,30 @@ struct ComputeBoundaryFlux {
 
   using inbox_tags = tmpl::list<FluxesTag>;
 
-  template <typename Flux, typename... Tags, typename... ArgumentTags>
-  static auto apply_flux(const Flux& flux,
-                         const tuples::TaggedTuple<Tags...>& self_data,
-                         const tuples::TaggedTuple<Tags...>& neighbor_data,
-                         tmpl::list<ArgumentTags...> /*meta*/) noexcept {
-    return flux(tuples::get<ArgumentTags>(self_data)...,
-                tuples::get<ArgumentTags>(neighbor_data)...);
+  template <typename... NormalDotFluxTags, typename... VariablesTags,
```

Comment:
make a new action for non-conservative systems and keep the current action for conservative systems